docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -0,0 +1,339 @@
+---
+title: "Using asset checks to check data freshness | Dagster Docs"
+description: "Use freshness checks to identify data assets that are overdue for an update."
+---
+
+# Using asset checks to check data freshness
+
+A freshness check is a type of [asset check](/concepts/assets/asset-checks) that allows you to define the amount of time between materializations/observations for Dagster [assets](/concepts/assets/software-defined-assets).
+
+Using freshness checks, you can easily identify assets that are overdue for an update. As freshness checks don't depend on a specific root cause, it makes them helpful in accounting for unknowns. For example, freshness checks can identify overdue assets caused by any of the following:
+
+- The pipeline hitting an error and failing
+- Runs were never scheduled
+- A backed up run queue
+- Runs are taking longer than expected to complete
+
+By the end of this guide, you'll understand what freshness checks are and how to implement them in your data pipelines.
+
+---
+
+## How it works
+
+An asset is considered fresh if it's been materialized or observed within a defined time window. Otherwise, assets are considered overdue and in need of an update. To identify these assets, you can use **freshness checks**.
+
+Let's say an asset begins materializing at 12:00AM and takes 10 minutes to complete. Allowing for operational constraints and delays, the asset should always be materialized by 12:30AM. Using a freshness check, we can define the time that the asset should be updated. If it's not, Dagster will mark it as **Overdue**.
+
+How freshness checks are implemented depends on the type of asset they target:
+
+- **For source assets**, which are assets whose data feeds a Dagster pipeline, freshness checks utilize the time the asset was last updated to determine freshness. In this guide, we'll demonstrate how to use [observable source assets](/concepts/assets/asset-observations#observable-source-assets) and a [schedule](/concepts/partitions-schedules-sensors/schedules) to achieve this.
```

Comment:
For both source assets and materializable assets, freshness checks utilize the time the asset was last updated. The difference is that the way "time the asset was last updated" is different for these two cases.